scripts/process_schemas.py

```diff
@@ -282,6 +282,9 @@ def __init__(self, args):
         string = "schemas-{0}".format(self.version[1:])
         self.schemaDir = os.path.join(self.tmpDir, string)
         self.avroJar = os.path.join(self.schemaDir, "avro-tools.jar")
+        if args.inputSchemasDirectory is not None:
+            self.schemaDir = args.inputSchemasDirectory
```

Comment:
I don't think we should work within the input directory tree, as this is potentially dangerous and can be confusing. I think we should leave self.schemaDir as it is, and instead copy the relevant files into the temporary directory. This way we're guaranteed to clean up properly after ourselves.